src/components/icon/documentation/Info.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { StoryHeader, StoryCode } from '../../../../.storybook/style/storybook-info.styles';
+
+const Info = (
+  <div>
+    <p>An Icon widget.</p>
+
+    <StoryHeader>How to use an Icon in a component:</StoryHeader>
+
+    <p>In your file</p>
+
+    <StoryCode padded>
+      {'import Icon from "carbon-react/lib/components/icon";'}
+    </StoryCode>
+
+    <p>To render an Icon:</p>
+
+    <StoryCode padded>
+      {'<Icon type="foo" />'}
+    </StoryCode>
+
+    <p>
+      {"'type' is a required prop"}
+    </p>
+
+    {/* eslint-disable-next-line */}
```

Comment:
is it necessary?

Reply:
It is, I got a long-line-length linting error. I did start re-writing it, but in the end, it made more sense to ignore the line rather than spend the time trying to get the character count down.